Chinese/dream/dream.md

```diff
@@ -0,0 +1,8 @@
+你來到這就是在作夢~ 來吧! 以下自由選擇:
+
+1. [我要穿越](../Yifan/modernCity.md)
+
+2. [我要當至尊寶](../lotus/%E5%90%83%E9%A5%BC%E5%B9%B2%E9%81%90%E6%83%B3.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.